src/pgrubic/rules/unsafe/US011.py

```diff
@@ -38,19 +40,22 @@ def visit_ColumnDef(
         """Visit ColumnDef."""
         if ancestors.find_nearest(ast.AlterTableCmd) and node.constraints:
             is_not_null = False
-            has_static_default = False
+            has_volatile_default = False
 
-            for constraint in node.constraints:
+            for constraint in typing.cast(tuple[ast.Constraint], node.constraints):
```

Comment:
Let's use variable declaration to type hints instead of the casting.

Reply:
Fixed by [f2fe1e2](https://github.com/bolajiwahab/pgrubic/pull/150/commits/f2fe1e22a8ac1fdf95630b8be45bbcddd3d62095)